test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -1042,7 +1041,20 @@ UCS_TEST_P(test_uct_cm_sockaddr, conn_to_non_exist_ip)
     ucs_status_t status;
     size_t size;
 
-    skip_tcp_sockcm();  /* error handling isn't implemented yet in tcp_sockcm  */
+    m_entities.clear();
+
+    /* tcp_sockcm requires setting this parameter to shorten the time of waiting
+     * for the connect() to fail when connecting to a non-existing ip.
+     * A transport for which this value is not configurable, like rdmacm,
+     * uct_config_modify will fail and have no effect. */
+    if (m_cm_config) {
+        /* coverity[check_return] */
+        uct_config_modify(m_cm_config, "SYN_CNT", "1");
+    }
+
```

Comment:
can we use modify_config()?

Reply:
yes, will change.
why though? many uct gtests use uct_config_modify.

Reply:
we ca change those as well.. but better use the common function in uct_test